.github/workflows/commit_artifacts.yml

```diff
@@ -122,23 +120,73 @@ jobs:
           mv build/oss-stable/react-refresh/cjs/react-refresh-babel.development.js \
             ./compiled/babel-plugin-react-refresh/index.js
 
+          ls -R ./compiled
+      - name: Move relevant files for React in fbsource into compiled
+        run: |
+          BASE_FOLDER='compiled/facebook-fbsource/xplat/js'
```

Comment:
do you mind making this a top level, sibling dir outside of `./compiled`, eg `./compiled-rn`? Reason being that the commit job just pushes the entire compiled dir to www and it'd be easier to separate them

Reply:
sure, I can change it. But it is already separated during commit step where only facebook-www folder is synced to react repo. The step for commit support `file_pattern`.